<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RosterSettings</class>
 <widget class="KittySDK::SettingPage" name="RosterSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>285</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Roster</string>
  </property>
  <property name="name" stdset="0">
   <string notr="true">Display.Roster</string>
  </property>
  <property name="icon" stdset="0">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="rosterGroupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QFormLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::ExpandingFieldsGrow</enum>
      </property>
      <item row="1" column="0" colspan="2">
       <widget class="QCheckBox" name="hideOfflineContactsCheckBox">
        <property name="text">
         <string>Hide offline contacts</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Status description:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string>Don't show</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Right side</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Under contact</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Avatars:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="comboBox_2">
        <item>
         <property name="text">
          <string>Don't show</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Right side</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Left side</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupsGroupBox">
     <property name="title">
      <string>Groups</string>
     </property>
     <layout class="QFormLayout" name="_2">
      <item row="0" column="0" colspan="2">
       <widget class="QCheckBox" name="groupsAsTabsCheckBox">
        <property name="text">
         <string>Display groups as tabs</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QCheckBox" name="groupsOnMainCheckBox">
        <property name="text">
         <string>Display groups on main tab</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="tipsGroupBox">
     <property name="title">
      <string>Tips</string>
     </property>
     <layout class="QFormLayout" name="_4">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::ExpandingFieldsGrow</enum>
      </property>
      <item row="1" column="0">
       <widget class="QLabel" name="tipDelayLabel">
        <property name="text">
         <string>Display tip after:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QCheckBox" name="tipsCheckBox">
        <property name="text">
         <string>Enable tips</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QCheckBox" name="displayTipsIPCheckBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Display IP</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="tipDelaySpinBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="suffix">
         <string> ms</string>
        </property>
        <property name="maximum">
         <number>2000</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="spacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KittySDK::SettingPage</class>
   <extends>QWidget</extends>
   <header>SDK/SettingPage.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>tipsCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>tipDelaySpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>63</x>
     <y>275</y>
    </hint>
    <hint type="destinationlabel">
     <x>133</x>
     <y>302</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tipsCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>displayTipsIPCheckBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>36</x>
     <y>285</y>
    </hint>
    <hint type="destinationlabel">
     <x>47</x>
     <y>328</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
